sanic/config.py

```diff
@@ -201,7 +201,11 @@ def load_environment_vars(self):
         for k, v in os.environ.items():
             if k.startswith(SANIC_PREFIX):
                 _, config_key = k.split(SANIC_PREFIX, 1)
-                if v.isdigit():
-                    self[config_key] = int(v)
+                # This is a float or an int
```

Comment:
\To me, the `isdigit` check is more explicit, but I guess if we want to convert bools to 0/1 as well the try/except will do that.